index.d.ts

```diff
@@ -0,0 +1,116 @@
+import { IncomingMessage, ServerResponse } from 'http';
+import { Http2ServerRequest, Http2ServerResponse } from 'http2';
+
+declare function Router<V extends Router.HTTPVersion = Router.HTTPVersion.V1>(
+  config?: Router.Config<V>
+): Router.Instance<V>;
+
+declare namespace Router {
+  enum HTTPVersion {
+    V1 = 'http1',
+    V2 = 'http2'
+  }
+
+  type HTTPMethod =
```

Comment:
We support all HTTP methods, as you can see [here](https://github.com/delvedor/find-my-way/blob/90348f3e4b92e1c52bc6be77b1e17b68eb5b5040/index.js#L488-L498).
This list probably needs to be updated.